macroid-core/src/main/scala/macroid/MultiEventTweaks.scala

```diff
@@ -0,0 +1,146 @@
+package macroid
+
+import scala.language.dynamics
+import scala.language.experimental.macros
+import android.view.View
+import scala.reflect.macros.{Context ⇒ MacroContext}
+import scala.language.postfixOps
+
+private[macroid] trait MultiEventTweaks {
+
+  import MultiEventTweakMacros._
+
+  type UnitHandler = String
+
+  type FuncHandler = String
+
+  sealed trait HandlerType
+
+  case object HUnit extends HandlerType
+
+  case object HUnitFunc extends HandlerType
+
+  object UnitOn extends Dynamic {
+    def applyDynamic[W <: View](event: String)(handler: Any): Tweak[W] = macro onUnitImpl[W]
+  }
```

Comment:
If I understand right, this is exactly the same as the current `On` and `FuncOn`, but delegating the implementation to the new multi-listener logic, right? I would prefer to avoid having duplicating functionality in the codebase. Perhaps you could just rewire the old objects to your implementation instead?